packages/bridge-controller/package.json

```diff
@@ -85,7 +85,7 @@
   },
   "peerDependencies": {
     "@metamask/accounts-controller": "^27.0.0",
-    "@metamask/assets-controllers": "^57.0.0",
+    "@metamask/assets-controllers": "^58.0.0",
```

Comment:
Nit: You can add a changelog entry for this bump under Changed section for the new version
```
- **BREAKING** Bump `@metamask/assets-controllers` peer dependency to `^58.0.0` ([#5672](https://github.com/MetaMask/core/pull/5672))